### DISCUSSION TITLE
exchange/tju-exchange/21-数学与应用数学-Andmilde-天大交换

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#51] [Announcements]
# exchange/tju-exchange/21-数学与应用数学-Andmilde-天大交换

A magical documentation generator.

https://hainanu-application.github.io/

**Comment by peng-yq** [comment]
准确来说，数据结构和c++没有太大关系，用了个引用

**Reply by SymphonicTechnicalProgressiveDeathcore** [reply]
行不更名，坐不改姓，本人回复，你的说法正确，如果从纯学科的角度来说，是对的

我强调的是课程安排的结果，因为天大是通过C++语言教数据结构的，所以如果没有C++基础，一定要注意到这一点，学起来要加倍努力

当然如果能提前自学更好，编程语言其实都是通的，会一种之后掌握第二种不会太难，核心的编程思想是一致的
